src/main/java/net/atos/client/zgw/zrc/ZRCClientService.java

```diff
@@ -136,6 +136,10 @@ public void deleteZaakInformatieobject(final UUID zaakInformatieobjectUuid, fina
         zrcClient.zaakinformatieobjectDelete(zaakInformatieobjectUuid);
     }
 
+    public void unlinkZaakInformatieobject(final UUID zaakInformatieobjectUuid, final String toelichting) {
```

Comment:
Verwijder deze mothode en pas maar de deleteZaakInformatieobject methode aan door er een extra string parameter aan toe te voegen: prefix. En dan een string.format bij het setten van de audittoelichting. 

Ik zie dat de deleteZaakInformatieobject methode gebruikt wordt voor ontkoppelen, verwijderen en verplaatsen. Verplaatsen voegt ook een prefix toe aan de toelichting, dit kun je dan refactoren dat het aan de deleteZaakInformatie methode wordt meegegeven. Dan dus op beide andere plekken voor ontkoppelen en verwijderen de deleteZaakInformatie methode aanroepen en de prefix meegeven.

Reply:
Opgelost in laatste commit - enum toegevoegd voor de drie verschillende prefixes